torch/serialization.py

```diff
@@ -17,7 +17,7 @@
 
 DEFAULT_PROTOCOL = 2
 
-LONG_SIZE = struct.Struct('=l').size
+LONG_SIZE = struct.Struct('=Q').size
```

Comment:
Is this equivalent?
According to the docs of `struct`, `l` stands for `long`, while `Q` is `unsigned long long`. Maybe you wanted to use `q` (`long long`) instead?

Reply:
Oh sorry, I' ll fix this